src/test/java/utilities/Util.java

```diff
@@ -14,5 +17,16 @@ public static void wait(int seconds) {
 			e.printStackTrace();
 		}
 	}
-
+	
+	/*
+	 * HomeWork2-7
+	 * 
+	 * @Author: claire zhang
+	 */
+	public static void targetLocation(WebDriver driver, String targetLocation) {
```

Comment:
1, The javadoc is meant to describe the function itself
2, The second parameter is better when it's a `By` object
3, Fix formating